### ISSUE TITLE
Attachments are not cloned when a transaction is duplicated (or copy/pasted)

### ISSUE THREAD

**Issue by whalley** [moneymanagerex/moneymanagerex#4183]
@whalley Attachments are not cloned when a transaction is duplicated (or copy/pasted)

I think they should be.  i.e. the duplicated transaction should be an exact clone

Thoughts?

_Originally posted by @tactilis in https://github.com/moneymanagerex/moneymanagerex/issues/4168#issuecomment-1030336026_

**Comment by Dandy64** [comment]
Well, I am not sure that this is a good idea. 
First of all, It is necessary to understand why we want to duplicate any transaction? Do we really want to create absolutely identical copy of the current transaction? Probably not. I suppose we intend to create a **similar** transaction because we do not want to fill some standard fields one more time.
OK, and what kind of attachments do we usually have? I think it can be bills or invoices, right? So I do not believe that a duplicate transaction should have the same invoice or bill as the original one.

**Comment by vomikan** [comment]
I agree with @Dandy64's opinion. In most cases, it is necessary to attach either a check or an invoice to the transaction. These are unique files identifying the actual transaction.

**Comment by whalley** [comment]
Some people have asked for it and some don't like the idea, guess it depends upon your workflow.

My current build also has the option to toggle this (with default off) which should make everyone happy...
![image](https://user-images.githubusercontent.com/487345/152697834-4c394f66-2940-4ba1-b406-10787c9f9bb9.png)

**Comment by Dandy64** [comment]
> My current build (which I'll release shortly) also has the option to toggle this (with default off) which should make everyone happy...

Yep, it is a very good decision making everyone happy! 👍
